### ISSUE TITLE
Machine template editor website is unreachable

### ISSUE THREAD

**Issue by FarFlyer** [tinkerspy/Automaton#89]
Currently, [Machine template editor](http://wolkendek.nl/atm/) is unreachable.

**Comment by tinkerspy** [comment]
Yeah. I moved to a tropical island, and while I took a miniaturized version of my server with me, my provider doesn't allow me to open up incoming ports. Unless someone else is able to host it (code is on github), the editor is down for the moment.

Rgdz, Tinkerspy

**Comment by FarFlyer** [comment]
Thanks for the quick response.

I hope you're enjoying the tropical island.

Best regards,

Davide Tagliafierro
Mechatronics Engineer
www.daveamps.it
Mobile: +39 329 067 0099

[from mobile]

Il ven 5 ago 2022, 18:37 Tinkerspy ***@***.***> ha scritto:

**Comment by dsteinfeld** [comment]
I've brought up the machine editor (at least for now) at: https://machine-editor.thesteinfelds.org.  All are welcome to use it!  A little payback for this terrific library.

**Comment by dsteinfeld** [comment]
@tinkerspy feel free to point https://wolkendek.nl/atm/ to https://machine-editor.thesteinfelds.org

**Comment by tinkerspy** [comment]
I pointed the wiki URL to your installation.

Thanks a lot!

**Comment by BlackJackMC** [comment]
[https://machine-editor.thesteinfelds.org/](https://machine-editor.thesteinfelds.org/) is now unreachable
